src/pages/index.js

```diff
@@ -5,96 +5,235 @@ import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import Section from "../components/Section.js";
 import UserInfo from "../components/UserInfo.js";
-import { initialCards, config } from "../utils/Constants.js";
+import { config } from "../utils/Constants.js";
+import Api from "../components/Api.js";
+import DeletePopup from "../components/DeletePopup.js";
+
+//Form const
+const addNewPlaceForm = document.forms["add-place-edit-fields"];
+const editAvatarForm = document.forms["edit-avatar-image"];
+const deleteCardForm = document.forms["delete-card-form"];
+const profileFormElement = document.forms["profile-edit-fields"];
+
+//Button consts
+const profileAddButton = document.querySelector(".profile__add-button");
+const profileEditButton = document.querySelector(".profile__edit-button");
+const editAvatarButton = document.querySelector(".profile__avatar-container");
 
 //Initializing cards
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: (data) => {
-      const cardElement = createCard(data);
-      cardSection.addItem(cardElement);
-    },
-  },
-  ".cards__list"
-);
 
-cardSection.renderItems();
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "a5df8bb7-ccf7-4af1-a820-819df810a6c4",
+    "Content-Type": "application/json",
+  },
+});
 
+Promise.all([api.getInitialCards(), api.getUserInfo()])
+  .then(([cards, userInfo]) => {
+    console.log(cards, userInfo);
+  })
+  .catch((err) => console.error(err));
+
+let cardSection;
+
+api
+  .getInitialCards()
+  .then((cards) => {
+    cardSection = new Section(
+      {
+        items: cards,
+        renderer: (data) => {
+          const cardElement = createCard(data);
+          cardSection.addItem(cardElement);
+        },
+      },
+      ".cards__list"
+    );
+    cardSection.renderItems();
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+//initializing user info
 const userInfo = new UserInfo({
   nameSelector: "#profile-title",
   descriptionSelector: "#profile-job",
+  avatarImage: "#profile-image",
 });
 
-//Initializing add place popup window
+api.getUserInfo().then((userData) => {
+  userInfo.setUserInfo(userData.name, userData.about);
+  userInfo.setUserAvatar(userData.avatar);
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**